test/html.test.js

```diff
@@ -930,3 +930,133 @@ function() {
     'active source buffers ends with audio source buffer');
 
 });
+
+QUnit.test('calls changeInfo_ on mediaSource when info is given for audio', function() {
```

Comment:
One is verifying that `changeInfo_` is always called when there is info on an audio segment. The other verifies that an event is triggered when the info of a track does actually change.